TASVideos/Extensions/FormFileExtensions.cs

```diff
@@ -55,7 +56,7 @@ public static void AddModelErrorIfOverSizeLimit(
 	{
 		if (!user.Has(PermissionTo.OverrideSubmissionConstraints) && movie.Length >= SiteGlobalConstants.MaximumMovieSize)
```

Comment:
This maximum movie size is going to be a problem for noscript users trying to upload text based formats like .fm2.  Not sure what to do about it though

Reply:
I mean if they're well-versed enough in technology that they care about disabling JS, they should be able to .zip a file before upload.